pkg/deploy/registry/deployconfig/etcd/etcd.go

```diff
@@ -46,3 +75,150 @@ func NewStorage(s storage.Interface) *REST {
 
 	return &REST{store}
 }
+
+// ScaleREST contains the REST storage for the Scale subresource of DeploymentConfigs.
+type ScaleREST struct {
+	registry     *deployconfig.Registry
```

Comment:
I don't think so, I think that's an artifact from copying from the Kubernetes code, which had it for some reason.  I'll take a look.